.devcontainer/devcontainer.json

```diff
@@ -39,7 +39,7 @@
         "omnisharp.useModernNet": true,
         "omnisharp.enableAsyncCompletion": true,
         // ms-dotnettools.csdevkit settings
-        "dotnet.defaultSolution": "Roslyn.sln",
+        "dotnet.defaultSolution": "Roslyn.slnx",
```

Comment:
I'm not expecting to see this change here. Is this merging some new changes from `main` into the feature branch?